safekeeper/src/handler.rs

```diff
@@ -72,8 +72,8 @@ impl postgres_backend::Handler for SafekeeperPostgresHandler {
     // ztenant id and ztimeline id are passed in connection string params
     fn startup(&mut self, _pgb: &mut PostgresBackend, sm: &FeStartupPacket) -> Result<()> {
         if let FeStartupPacket::StartupMessage { params, .. } = sm {
-            self.ztenantid = match params.get("ztenantid") {
-                Some(z) => Some(ZTenantId::from_str(z)?), // just curious, can I do that from .map?
+            self.tenantid = match params.get("tenantid") {
```

Comment:
It will be caught by many tests. It is set in walproposer.c like `options='-c ztimelineid=%s...`

Reply:
Yes thank you @arssher. I've raised a pull-request for walprosper.c [here](https://github.com/neondatabase/postgres/pull/168).

Reply:
Thanks. You should point vendor/postgres in this PR to your `[phoenix24:rename-tenantid](https://github.com/phoenix24/postgres/tree/rename-tenantid)` branch, otherwise tests won't pass.

Reply:
thanks @arssher I actually have pointed current pr to related [postgres changes](https://github.com/neondatabase/postgres/pull/168)., these below tests fail due to permission failure with uploading failure on circle-ci.